modules/sql/src/main/scala/EnumDef.scala

```diff
@@ -49,11 +54,14 @@ object EnumDef {
     implicit val caseDouble       = at[Double  ](a => a.toString)
     implicit val caseDuration     = at[Duration](a => s"java.time.Duration.ofMillis(${a.toMillis})")
     implicit val caseArcseconds   = at[Arcseconds](a => s"gem.math.Angle.fromDoubleArcseconds(${a.toArcsecs})")
-    implicit val caseDegrees      = at[Degrees](a => s"gem.math.Angle.fromDoubleDegrees(${a.toDegrees})")
+    implicit val caseDegrees      = at[Degrees ](a => s"gem.math.Angle.fromDoubleDegrees(${a.toDegrees})")
     implicit val caseZoneId       = at[ZoneId  ](a => s"""java.time.ZoneId.of("${a.toString}")""")
 
-    implicit val caseWavelengthNm = at[Wavelength.Nm ](a => s"gem.math.Wavelength.unsafeFromAngstroms(${a.toAngstrom})")
-    implicit val caseWavelengthUm = at[Wavelength.Um ](a => s"gem.math.Wavelength.unsafeFromAngstroms(${a.toAngstrom})")
+    implicit val caseWavelengthNm = at[Wavelength.Nm](a => s"gem.math.Wavelength.unsafeFromAngstroms(${a.toAngstrom})")
+    implicit val caseWavelengthUm = at[Wavelength.Um](a => s"gem.math.Wavelength.unsafeFromAngstroms(${a.toAngstrom})")
+    implicit val caseSlitWidthNm  = at[SlitWidth.Nm ](a => s"gem.math.Wavelength.unsafeFromAngstroms(${a.toAngstrom})")
+    implicit val caseSlitCenterNm = at[SlitCenter.Nm](a => s"gem.math.Wavelength.unsafeFromAngstroms(${a.toAngstrom})")
```

Comment:
Do you think it's worth having a separate Slit dimensions class in `math`? I'm thinking SlitWidth and SlitCenter are just placeholders that eventually need to be replaced by a more general solution for units with alternative representations.

Reply:
I think if we were using Libra slit widths would just be nm.  `Wavelength` doesn't really make sense here but it's probably not worth having a `SlitWidth` type?  Until then I wouldn't look askance at just using an `Int` and "knowing" it is nm... We do that type of thing in other places.

Reply:
i thought we had discarded no use Libra, is that being reconsidered?

Reply:
I didn't think it was discarded, but rather under-evaluated.  I had some concerns about it but I also have some concerns about the status quo.

Reply:
By status quo do you mean using `Int` or `squants`?

Reply:
My concern is that it's new and adopting it would be a "deep" dependency that would infect the lowest parts of the model (i.e., removing it later would be hard). I'm generally positive, but we will probably need to make improvements to it first. In any case I think Someone™ needs to take some time and evaluate it properly.

Reply:
I mean using primitives with no unit information as we do in a few places.  The problem with Squants, I think, is that it stores values as a `Double`:

```scala
final class Length private (val value: Double, val unit: LengthUnit)
```

Reply:
Right, forgot about that. I was meaning to progress on generalizing that for squants 2.0. Mabye @arturog8m wants to support that work